apps/api/src/app/integrations/usecases/deactivate-integration/deactivate-integration.usecase.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { IntegrationRepository } from '@notifire/dal';
+import { DeactivateIntegrationCommand } from './deactivate-integration.command';
+
+@Injectable()
+export class DeactivateIntegration {
```

Comment:
```suggestion
export class DeactivateSmiliarChannelIntegrations {
```

Maybe better to use a name like this to better portray what it does? And then we can remove the comment on line 8 that was needed. Wdyt?

Reply:
Haha you're right!  I just was a bit afraid of the long name 😅